Storage/DataManager.php

```diff
@@ -106,6 +106,15 @@ public function load(FormFlowInterface $flow) {
 	/**
 	 * {@inheritDoc}
 	 */
+	public function exists(FormFlowInterface $flow) {
+		$savedFlows = $this->storage->get(DataManagerInterface::STORAGE_ROOT, array());
+		// A flow exists if the instance ID exists in the saved flows.
```

Comment:
Considering [this comment](https://github.com/craue/CraueFormFlowBundle/pull/220#discussion-diff-55337490), I still think this one can be removed.